### PR TITLE
doc: Updating AWS install guides for terraform release.

### PR DESCRIPTION
**Pull request by zbwright**
Given the time constraints, I wanted to get this process started. I have not yet QA'd the command line process for these.

### CODE REVIEW

File: Documentation/install/aws/requirements.md [hunk 1]
```diff
@@ -126,27 +126,28 @@ The following table includes the high level networking features required to inst
...
 ### Using an existing VPC
 
 By default, Tectonic Installer creates a new AWS Virtual Private Cloud (VPC) for each cluster. Advanced users can choose to use an existing VPC instead. An existing VPC must have an [Internet Gateway][aws-vpc-inet-gateway]. Tectonic Installer will not create an Internet Gateway in an existing VPC.
 
 An existing VPC for a public cluster must have a public subnet for controllers, and a private subnet for workers. An existing VPC for an internal cluster must have 2 private subnets, one each for controllers and workers.
 
-Public subnets have a default route to the Internet Gateway and should auto-assign IP addresses. Private subnets have a default route to a default gateway, e.g. a NAT Gateway, a Virtual Private Gateway etc.
+Public subnets have a default route to the Internet Gateway and should auto-assign IP addresses. Private subnets have a default route to a default gateway, such as a NAT Gateway or a Virtual Private Gateway.
+
+*DHCP Options Set* attached to the VPC must have AWS [private domain name][aws-vpc-dns-hostnames]. In us-east-1 region AWS private domain name is ec2.internal whereas other regions use region.compute.internal.
```
Comment (joshix) [inline Documentation/install/aws/requirements.md]: _an_ AWS private domain name, perhaps?

File: Documentation/install/aws/aws-terraform.md [hunk 2]
```diff
@@ -8,65 +8,76 @@ Generally, the AWS platform templates adhere to the standards defined by the pro
...
+$ curl -O https://releases.tectonic.com/tectonic-1.6.2-tectonic.1.tar.gz
 ```
 
-Initialize the TerraForm configuration with Installer's location and export the path to that configuration:
+Then, extract the tarball, and launch the Tectonic Installer:
 
+```bash
+tar xzvf tectonic-1.6.2-tectonic.1.tar.gz # to extract the tarball
+$ cd tectonic/tectonic-installer # to change to the previously untarred directory
+$ ./$PLATFORM/installer # to run the Tectonic Installer
```
Comment (Quentin-M) [inline Documentation/install/aws/aws-terraform.md]: Is that what we typically document? They could download/extract/run all using point&click though?

Comment (Quentin-M) [inline Documentation/install/aws/aws-terraform.md]: If this is a document describing how to use the installer using the CLI, they don't need to run the GUI at all.

Comment (robszumski) [inline Documentation/install/aws/aws-terraform.md]: Oh right, this is a good point.

Comment (zbwright) [inline Documentation/install/aws/aws-terraform.md]: I thought the whole point of the installer was that it was launched and accessible in a browser. I also thought the move to this installer was to make setup easier for the less technical people, who could progress from easy GUI to full CLI use as they wish. Is there a way to run through the installer using the CLI only?

Comment (zbwright) [inline Documentation/install/aws/aws-terraform.md]: actually, with my better understanding now, let me edit this a little.

Comment (Quentin-M) [inline Documentation/install/aws/aws-terraform.md]: Discussed offline.

**Comment by Quentin-M** [comment]
Change looks good to me! Thanks Beth!

**Comment by Quentin-M** [comment]
@zbwright Could you please rebase and we'll merge?

**Comment by zbwright** [comment]
I was told make was going away. Is this addition accurate?

**Comment by robszumski** [comment]
@zbwright I didn't see any more references to `make`. Are there any still?

**Comment by zbwright** [comment]
@rithujohn191 probably wasn't aware that this doc was in progress, and added the following lines (which I discovered when trying to resolve conflicts):

+ - **Make, go, yarn** - This guide uses `make`, `go` and `yarn` to build the Tectonic Installer.

maybe go and yarn are required now? but this was the conflict.

**Comment by robszumski** [comment]
@zbwright Yes, yarn, make etc are all required for the dev/contributor workflow, but not for end users after this release.

**Pull request merged**